### PR TITLE
[access-50] company details

### PR DESCRIPTION
**Pull request by cjestal**

### CODE REVIEW

File: resources/views/admin/sidebar.blade.php [hunk 1]
```diff
@@ -41,6 +41,11 @@
                         Purchase Orders
                     </a>
                 </li>
+                <li role="presentation">
+                    <a href="{{ url('/company') }}">
```
Comment (rosoarv) [inline resources/views/admin/sidebar.blade.php]: use 'company-details' for the uri.

File: resources/views/company-details/form.blade.php [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        {!! Form::text('state_province', null, ['class' => 'form-control']) !!}
+        {!! $errors->first('state_province', '<p class="help-block">:message</p>') !!}
+    </div>
+</div>
+
+<div class="form-group {{ $errors->has('country') ? 'has-error' : ''}}">
+    {!! Form::label('country', 'Country', ['class' => 'col-md-4 control-label']) !!}
+    <div class="col-md-6">
+        {!! Form::select('country', ['Philippines', 'China', 'Japan'], 'Philippines', ['class' => 'form-control']) !!}
+        {!! $errors->first('country', '<p class="help-block">:message</p>') !!}
```
Comment (rosoarv) [inline resources/views/company-details/form.blade.php]: start first with country -> state / province -> city

File: resources/views/company-details/form.blade.php [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    {!! Form::label('email', 'Email', ['class' => 'col-md-4 control-label']) !!}
+    <div class="col-md-6">
+        {!! Form::email('email', null, ['class' => 'form-control']) !!}
+        {!! $errors->first('email', '<p class="help-block">:message</p>') !!}
+    </div>
+</div>
+
+<div class="form-group {{ $errors->has('url') ? 'has-error' : ''}}">
+    {!! Form::label('url', 'Url', ['class' => 'col-md-4 control-label']) !!}
+    <div class="col-md-6">
```
Comment (rosoarv) [inline resources/views/company-details/form.blade.php]: use "Website"

File: resources/views/company-details/form.blade.php [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+        {!! Form::text('url', null, ['class' => 'form-control']) !!}
+        {!! $errors->first('url', '<p class="help-block">:message</p>') !!}
+    </div>
+</div>
+
+<div class="form-group {{ $errors->has('tel_no') ? 'has-error' : ''}}">
+    {!! Form::label('tel_no', 'Tel No', ['class' => 'col-md-4 control-label']) !!}
+    <div class="col-md-6">
```
Comment (rosoarv) [inline resources/views/company-details/form.blade.php]: use "Telephone No." and change form type to text.

File: resources/views/company-details/form.blade.php [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+        {!! Form::number('tel_no', null, ['class' => 'form-control']) !!}
+        {!! $errors->first('tel_no', '<p class="help-block">:message</p>') !!}
+    </div>
+</div>
+
+<div class="form-group {{ $errors->has('mob_no') ? 'has-error' : ''}}">
+    {!! Form::label('mob_no', 'Mob No', ['class' => 'col-md-4 control-label']) !!}
+    <div class="col-md-6">
+        {!! Form::number('mob_no', null, ['class' => 'form-control']) !!}
```
Comment (rosoarv) [inline resources/views/company-details/form.blade.php]: use "Mobile No." and change form type to text.

File: resources/views/company-details/form.blade.php [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+        {!! $errors->first('mob_no', '<p class="help-block">:message</p>') !!}
+    </div>
+</div>
+
+<div class="form-group {{ $errors->has('fax_no') ? 'has-error' : ''}}">
+    {!! Form::label('fax_no', 'Fax No', ['class' => 'col-md-4 control-label']) !!}
```
Comment (rosoarv) [inline resources/views/company-details/form.blade.php]: period in "No" in "Fax No" and change form type to text.

File: routes/web.php [hunk 7]
```diff
@@ -31,3 +31,5 @@
 
 
 Route::resource('warehouse/inventory-manage', 'Warehouse\\InventoryManageController');
+
+Route::resource('company', 'CompanyDetails\\CompanyDetailsController');
```
Comment (rosoarv) [inline routes/web.php]: use route "company-details"

**Review by rosoarv — CHANGES_REQUESTED** [review]

File: resources/views/company-details/form.blade.php (hunk 3 above)
Comment (cjestal) [reply resources/views/company-details/form.blade.php]: use "Website" where?

Comment (rosoarv) [reply resources/views/company-details/form.blade.php]: "Website" as label instead of "Url"

**Comment by cjestal** [comment]
@rosoarv please see my corrections

**Review by rosoarv — APPROVED** [review]

**Review by rosoarv — APPROVED** [review]

**Comment by rosoarv** [comment]
@cjestal great. btw, define or describe the changes you made to the code as the commit message. Anyways, this looks good. Please merge to master. Thanks!

**Comment by cjestal** [comment]
@rosoarv noted on the commit messages.

**Pull request merged**